examples/batch/argo-workflows/helm-charts/seldon-batch-workflow/templates/workflow.yaml

```diff
@@ -0,0 +1,145 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  {{- if eq .Values.workflow.useNameAsGenerateName false }}
+  name: {{ .Values.workflow.name }}
+  {{- else }}
+  generateName: {{ .Values.workflow.name }}
+  {{- end }}
+  namespace: {{ .Values.workflow.namespace }}
```

Comment:
What do you mean by release.namespace?

Reply:
Do you mean it being expected to deploy in the same namespace as the workflow?

Reply:
I mean so that it receives the value of ‘helm template -n <NAMESPACE>’, which can be used in templates as ‘.Release.Namespace’.

Or do you think it should live on a different namespace?

Reply:
Oh I see! Ok I understand now - so currently the way that it's working is with helm template instead of helm install, although I'd assume that would still be namespaced bound. In regards to the namespace expectation, I was thinking there may be a reason where the person running it may want to run the seldon deployment in a different namespace to the workflow, although I think this could as well be a constraint we may want to impose. I guess at this point this would be more of a "sample/example" helm chart, so we could see how people use it and based on that impose these restrictions on the "supported" helm chart we release

Reply:
Got it, makes sense! :+1: